src/Schema/Exception/TypeMismatch.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAPIValidation\Schema\Exception;
+
+use Exception;
+use function gettype;
+use function sprintf;
+
+class TypeMismatch extends Exception
+{
+    /**
+     * @param mixed $value
```

Comment:
Do we really need `mixed` typehint everywhere? It does not bring any new info

Reply:
Good question. Doctrine REQUIRES us to provide at least something about "untyped" values. It would be better to provide a type, but there can be anything, actually. So If I remove this, doctrine checks fail...

Reply:
Ehh, okay